whois-commons/src/main/resources/nrtm_schema.sql

```diff
@@ -94,6 +94,7 @@ CREATE TABLE `key_pair`
     `public_key`  VARBINARY(3000) NOT NULL,
     `created`     bigint unsigned NOT NULL,
     `expires`     bigint unsigned NOT NULL,
+    `isActive`    bit(1) NOT NULL,
```

Comment:
Can you rename to `is_active` to be consistent with how the other columns are named?